### ISSUE TITLE
ImportError on Linux: rotate-backups will not run

### ISSUE THREAD

**Issue by am97** [xolox/python-rotate-backups#24]
Hi,

I installed rotate-backups on my Fedora 31 with

```
pip install --user rotate-backups
```

but I get an ImportError when trying to run rotate-backups from the CLI, for example:

```
am@am:~$ rotate-backups -h
Traceback (most recent call last):
  File "/home/am/.local/bin/rotate-backups", line 5, in <module>
    from rotate_backups.cli import main
  File "/home/am/.local/lib/python3.7/site-packages/rotate_backups/cli.py", line 211, in <module>
    from coloredlogs import WINDOWS
ImportError: cannot import name 'WINDOWS' from 'coloredlogs' (/home/am/.local/lib/python3.7/site-packages/coloredlogs/__init__.py)
```

The problem seems to have been introduced by [this commit](https://github.com/xolox/python-rotate-backups/commit/ed1c56c4fefb575922082df7c676a6536f864619) 4 days ago.

If I open a python3 console, I can import `coloredlogs` without problem, but can't import `WINDOWS` from `coloredlogs`

```
am@am:~$ python3
Python 3.7.6 (default, Jan 30 2020, 09:44:41) 
[GCC 9.2.1 20190827 (Red Hat 9.2.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import coloredlogs
>>> from coloredlogs import WINDOWS
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'WINDOWS' from 'coloredlogs' (/home/am/.local/lib/python3.7/site-packages/coloredlogs/__init__.py)
```

**Comment by xolox** [comment]
Hi @am97 and thanks for reporting this bug. I've just released [rotate-backups 8.0](https://rotate-backups.readthedocs.io/en/latest/changelog.html#release-8-0-2020-02-18) which fixes this issue.

**Comment by am97** [comment]
Thanks !
